website/src/components/BlogsSlider/style.ts

```diff
@@ -10,7 +10,19 @@ const useStyles = makeStyles((theme: Theme) => ({
   media: {
     height: 0,
     paddingTop: "56.25%",
-    borderRadius: '24px 24px 24px 0px',
+    borderRadius: "24px 24px 24px 0px",
+    position: 'relative',
+    overflow: 'hidden',
+    '& img':{
+      minWidth: '100%',
+      minHeight: '100%',
+      maxWidth: '100%',
+      position: 'absolute',
+      left: '0',
+      top: '0',
+      objectFit: 'cover',
+      objectPosition: 'left'
```

Comment:
why `left`, why not `middle`

Reply:
It didn't look good when i tried with `center`